app/models/notification_policy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: notification_policies
+#
+#  id                      :bigint(8)        not null, primary key
+#  account_id              :bigint(8)        not null
+#  filter_not_following    :boolean          default(FALSE), not null
+#  filter_not_followers    :boolean          default(FALSE), not null
+#  filter_new_accounts     :boolean          default(FALSE), not null
+#  filter_private_mentions :boolean          default(TRUE), not null
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#
+
+class NotificationPolicy < ApplicationRecord
+  belongs_to :account
+
+  attr_reader :pending_requests_count, :pending_notifications_count
+
+  def summarize!
+    @pending_requests_count, @pending_notifications_count = NotificationRequest.where(account: account, dismissed: false).limit(100).pick(Arel.sql('count(*), coalesce(sum(notifications_count), 0)::bigint'))
```

Comment:
Two ideas here:

- Pull out the entire `model.where.pick...` chain to private method?
- Instead of the multiple var assignment, do separate assignments on diff lines and use `.first` and `.last` results of that extracted (and memoized) private method?

Separately - is the `100` here enforcing a sort of default limit on max number to appear in a summary/count? Maybe pull that out to more intention-revealing constant name in this class?